docs/_config.yml

```diff
@@ -24,7 +24,7 @@ description: >- # this means to ignore newlines until "baseurl:"
   This site contains links to Dafny documentation.
   The most significant is the Dafny Reference Manual.
 baseurl: "" # the subpath of your site, e.g. /blog
-url: "https://dafny-lang.github.io/dafny" # the base hostname & protocol for your site, e.g. http://example.com
+url: "https://dafny-lang.github.io" # the base hostname & protocol for your site, e.g. http://example.com
```

Comment:
```suggestion
url: "https://dafny.org" # the base hostname & protocol for your site, e.g. http://example.com
```

Reply:
The Banner is a png, so it is not easy to change. I'd just leave it be.
I prefer to fix the URL in a later PR when we are actually doing the switch from dafny-lang.github.io, which this PR does not do -- and I'll work on once this PR is agreed to.

Reply:
You're right, adjusting the background colour in the css is easier, but this turned out decent as well. We might want to update it with some VS Code UI later anyway.
![image](https://user-images.githubusercontent.com/143799/180467360-131a8927-19d4-47d8-8dbb-27eb87b6f3d3.png)

Reply:
Good call in the url, but keep it `https://dafny-lang.github.io/dafny` then.

Reply:
In general I think any change we make should be a strict improvement over the status quo.